module04/01-Cash-Desk/solution.py

```diff
@@ -0,0 +1,98 @@
+class Bill:
```

Comment:
You can try creating classes that inherit from `object` e.g. `SomeExample(object):` this is the new style class